docs/BRANCH.md

```diff
@@ -0,0 +1,61 @@
+# Branch Strategy
+
+## Branches
+
+| Branch                                     | Description                                                                                                                                                              |
+| ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| main                                       | Production branch.                                                                                                                                                       |
+| issue/#{issue-number}                      | Used to work on an issue.                                                                                                                                                |
+| feature/{name}                             | Used to work on feature development other than issues.                                                                                                                   |
```

Comment:
![imo-badge](https://img.shields.io/badge/review-imo-skyblue.svg) 

I thought it might be confusing since it is `feature/#1` on the Mermaid graph.

Reply:
@Yamasaki-pan961 
Thank you!
Fixed https://github.com/yumemi-inc/flutter-yumemi-lints/pull/107/commits/54cbf3391d1c306b13a9fc2d0e6a62fc85be36d4